### DISCUSSION TITLE
support cookiecutter

### DISCUSSION THREAD

**Discussion by naortalmor1** [aquasecurity/trivy#4803] [Ideas]
### IDs

AVD-DS-0001 (DS001)

### Description

Using Jinja template and Cookiecutter to define Dockerfile to use a base image, trivy incorrectly identifies the tag as missing and identifies "latest". 

### Reproduction Steps

```bash
1. clone this repository - https://github.com/naortest1/cookiecutter-data-experimentation
2. Run trivy config scan
3. Issue found - Specify a tag in the 'FROM' statement for image '{{cookiecutter.base_image}}'
```

### Target

Filesystem

### Scanner

Misconfiguration

### Debug Output

```bash
Not relevant
```

### Version

```bash
0.43.1
```

### Checklist

- [X] Read [the documentation regarding wrong detection](https://aquasecurity.github.io/trivy/dev/community/contribute/discussion/#false-detection)
- [X] Ran Trivy with `-f json` that shows data sources and confirmed that the security advisory in data sources was correct

**Comment by itaysk** [comment]
Trivy doesn't support template files (or Jinga), it expects this Dockerfile to be valid which is not. 
Am I correct to reclassify this as feature request? or you expected a different behavior from trivy

**Reply by itaysk** [reply]
based on offline discussion I converted this to "Idea" and renamed the discussion
